lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -39,14 +39,14 @@
         description:
             - App service plan. Required for creation.
             - It can be name of existing app service plan in same resource group as web app.
-            - "It can be resource id of existing app service plan. eg.,
-              /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>"
-            - It can be a dict which contains C(name), C(resource_group), C(sku), C(is_linux) and C(number_of_workers).
-            - C(name). Name of app service plan.
-            - C(resource_group). Resource group name of app service plan.
-            - C(sku). SKU of app service plan. For allowed sku, please refer to U(https://azure.microsoft.com/en-us/pricing/details/app-service/linux/).
-            - C(is_linux). Indicates Linux app service plan. type bool. default False.
-            - C(number_of_workers). Number of workers.
+            - It can be resource ID of existing app service plan. for example
+              /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>.
+            - It can be a dict which contains I(name), I(resource_group), I(sku), I(is_linux) and I(number_of_workers).
```

Comment:
Are all five parameters required for the dict to work?
```suggestion
            - Can be a dict containing five parameters, defined below.
```